validator_manager/src/list_validators.rs

```diff
@@ -0,0 +1,203 @@
+use clap::{Arg, ArgAction, ArgMatches, Command};
+use eth2::SensitiveUrl;
+use serde::{Deserialize, Serialize};
+use std::path::PathBuf;
+
+use crate::{common::vc_http_client, DumpConfig};
+
+pub const CMD: &str = "list";
+pub const VC_URL_FLAG: &str = "vc-url";
+pub const VC_TOKEN_FLAG: &str = "vc-token";
+
+pub fn cli_app() -> Command {
+    Command::new(CMD)
+        .about("Lists all validators in a validator client using the HTTP API.")
+        .arg(
+            Arg::new(VC_URL_FLAG)
+                .long(VC_URL_FLAG)
+                .value_name("HTTP_ADDRESS")
+                .help(
+                    "A HTTP(S) address of a validator client using the keymanager-API. \
+            If this value is not supplied then a 'dry run' will be conducted where \
+            no changes are made to the validator client.",
```

Comment:
I removed this description in [e5a618c](https://github.com/sigp/lighthouse/pull/6261/commits/e5a618c24f0dc9b5f93f6d0d54cb0004197cb012) for all `import`, `list`, and `delete`. Because from testing, without supplying `vc-url` flag, it still able to make changes to the VC, using the default VC HTTP url.